install/helm/agones/templates/crds/_gameserverspecschema.yaml

```diff
@@ -153,6 +153,42 @@ properties:
             type: integer
             title: The initial player capacity of this Game Server
             minimum: 0
+{{- if .featureCountsAndLists }}
+      counters:
+        type: array
+        title: Array of player, room, session, etc. counters
+        nullable: true
+        maxItems: 1000
+        items:
+          type: object
+          properties:
```

Comment:
So in the original design we have this:

```yaml
  counters: # list of counters. Key value below is the key for each counter.
    rooms: # key for the counter (room)
      default: 1 # initial value
      capacity: 100 # maximum possible count value
```

Where the name is the key, and `default` is the default value.

In this type of setup, what is the key used for? And why the deviation from the design?

Reply:
I haven't put in the defaults yet, so I'll add those in in one or two PRs. Along those lines, would it make sense for the default to be 0 for a counter initial value?

Reply:
My point is that the naming in this schema doesn't match the design.